src/main/java/com/nicoislost/inputs/KeyBinds.java

```diff
@@ -1,31 +1,40 @@
 package com.nicoislost.inputs;
 
+import com.google.common.collect.ImmutableList;
 import net.fabricmc.fabric.api.client.keybinding.v1.KeyBindingHelper;
 import net.minecraft.client.option.KeyBinding;
 import net.minecraft.client.util.InputUtil;
 import org.lwjgl.glfw.GLFW;
 
+import java.util.List;
+
 public class KeyBinds {
-    public static KeyBinding keyBinding1;
-    public static KeyBinding keyBinding2;
-    public static KeyBinding keyBinding3;
-    public static void RegisterKeyBinds() {
-        keyBinding1 = KeyBindingHelper.registerKeyBinding(new KeyBinding(
-                "key.nicoislost.zoom1",
-                InputUtil.Type.KEYSYM,
-                GLFW.GLFW_KEY_C, "category.nicoislost.zoom"
-        ));
 
-        keyBinding2 = KeyBindingHelper.registerKeyBinding(new KeyBinding(
-                "key.nicoislost.zoom2",
-                InputUtil.Type.KEYSYM,
-                GLFW.GLFW_KEY_UNKNOWN, "category.nicoislost.zoom"
-        ));
+    public static final KeyBinding KEY_BINDING_1 = KeyBindingHelper.registerKeyBinding(new KeyBinding(
+            "key.zoom-o-matic.zoom1",
+            InputUtil.Type.KEYSYM,
+            GLFW.GLFW_KEY_C, "category.zoom-o-matic.zoom"
+    ));
+    public static final KeyBinding KEY_BINDING_2 = KeyBindingHelper.registerKeyBinding(new KeyBinding(
+            "key.zoom-o-matic.zoom2",
+            InputUtil.Type.KEYSYM,
+            GLFW.GLFW_KEY_UNKNOWN, "category.zoom-o-matic.zoom"
+    ));
+    public static final KeyBinding KEY_BINDING_3 = KeyBindingHelper.registerKeyBinding(new KeyBinding(
+            "key.zoom-o-matic.zoom3",
+            InputUtil.Type.KEYSYM,
+            GLFW.GLFW_KEY_UNKNOWN, "category.zoom-o-matic.zoom"
+    ));
+
+    private static final ImmutableList<KeyBinding> keyBindings = ImmutableList.of(
+            KEY_BINDING_1,
+            KEY_BINDING_2,
+            KEY_BINDING_3
+    );
+
```

Comment:
You could get this via using a `enum` for free ;)

Reply:
Completely agree, i wrote all of that at like 3am i think so eeee.

Reply:
By no means a criticism, just an observation. There are pros and cons to both approaches - needs drive musts 👍